sdk/eventhub/event-hubs/perf-tests.yml

```diff
@@ -25,12 +25,12 @@ PackageVersions:
   '@azure/logger': source
 
 Tests:
-- Test: send
+- Test: send-events-batch
   Class: SendTest
   Arguments:
   - --event-size 1024 --batch-size 100 --parallel 64
 
-- Test: subscribe
+- Test: receive-events
```

Comment:
Nit: On second thought, I feel like we should also have the `-batch` suffix here for a bit more clarity and consistency, especially since looking at the SB tests, we test both receive in batch/or non-batch form. In Python we have EH perf-tests for receive in batchs/non-batches, as well. Not a blocker, and ultimately doesn't matter too much, but WDYT?

Reply:
`receive-queue-messages-batch` in SB is different from `receive-events` here.

```js
// receive-queue-messages-batch in SB
messages = await receiver.receiveMessages(...)
// (= Batch)

// receive-queue-messages in SB
    receiver.subscribe(
      {
        processMessage: async (_message: ServiceBusReceivedMessage) => {
          .....
        },
        processError: async (args: ProcessErrorArgs) => {
          ....
        },
      }
    );
```
```js
// receive-events in EH
    receiver.subscribe(
      {
        processEvents: async (events: ReceivedEventData[], _context: PartitionContext) => {
          // ..... events (= Batch)
        },
        processError: async (error: Error | MessagingError, _context: PartitionContext) => { 
          .....
        },
      }
    );
```
Adding batch to EH streaming might be misleading in the context of EH vs SB.


---------------------

If you want to put "batch" in the name, I might probably prefer..
1. `receive-queue-messages-batch` in SB 
2. `process-queue-messages` in SB
3. `process-events-batch` in EH

Reply:
That works for me :)